### PR TITLE
Fix tests.

### PR DESCRIPTION
**Pull request by floitsch**
Some gold files were out of sync.

The refactoring of the configuration made the test flags not work anymore.

### CODE REVIEW

File: cmd/toitpkg/main.go [hunk 1]
```diff
@@ -47,24 +40,22 @@ var (
...
-	rootCmd.Flags().MarkHidden("config")
-	rootCmd.PersistentFlags().StringVar(&cacheDir, "cache", "", "cache dir")
-	rootCmd.Flags().MarkHidden("cache")
-	rootCmd.PersistentFlags().BoolVar(&noDefaultRegistry, "no-default-registry", false, "Don't use default registry if none exists")
-	rootCmd.Flags().MarkHidden("no-default-registry")
-	rootCmd.PersistentFlags().BoolVar(&noAutosync, "no-autosync", false, "Don't automatically sync")
-	rootCmd.Flags().MarkHidden("no-autosync")
-	rootCmd.PersistentFlags().BoolVar(&shouldPrintTracking, "track", false, "Print tracking information")
-	rootCmd.Flags().MarkHidden("track")
+	cfgFile, _ := os.LookupEnv("TOIT_CONFIG_FILE")
```
Comment (jstroem) [inline cmd/toitpkg/main.go:43]: use `os.Genenv`

File: cmd/toitpkg/main.go [hunk 2]
```diff
@@ -47,24 +40,22 @@ var (
...
+	cacheDir, _ := os.LookupEnv("TOIT_CACHE_DIR")
+	noDefaultRegistry, _ := os.LookupEnv("TOIT_NO_DEFAULT_REGISTRY")
+	shouldPrintTracking, _ := os.LookupEnv("TOIT_SHOULD_PRINT_TRACKING")
+	sdkVersion, _ := os.LookupEnv("TOIT_SDK_VERSION")
+	noAutosync, _ := os.LookupEnv("TOIT_NO_AUTO_SYNC")
 
-	rootCmd.PersistentFlags().StringVar(&sdkVersion, "sdk-version", "", "The SDK version")
+	cfgFile = strings.Trim(cfgFile, " ")
```
Comment (jstroem) [inline cmd/toitpkg/main.go:50]: use `strings.TrimSpace`

**Review by jstroem — APPROVED** [review]

File: cmd/toitpkg/main.go (hunk 1 above)
Comment (floitsch) [reply cmd/toitpkg/main.go]: done.

File: cmd/toitpkg/main.go (hunk 2 above)
Comment (floitsch) [reply cmd/toitpkg/main.go]: done.

Using a function now (`getTrimmedEnv`).

**Pull request merged**